app/templates/components/nypr-svg/jlgreene-logo.hbs

```diff
@@ -0,0 +1,11 @@
+  <path d="M215.3442,30.8857 C215.3442,22.9217 217.9652,16.0177 230.1622,16.0177 C242.7122,16.0177 243.1152,24.3837 242.8642,33.9097 L223.0052,33.9097 C223.4582,37.3367 225.1222,39.3527 230.2132,39.3527 C234.5982,39.3527 237.0672,38.2947 239.1842,36.7327 L243.1662,41.8737 C239.4872,44.6457 235.8582,45.7537 230.2132,45.7537 C217.8642,45.7537 215.3442,38.9997 215.3442,30.8857 M223.0052,28.2647 L235.6562,28.2647 C235.4542,24.9897 234.7992,22.6197 229.9102,22.6197 C225.0222,22.6197 223.4082,24.7867 223.0052,28.2647" id="e2" stroke="none" fill="#221F20" fill-rule="evenodd"></path>
+  <path d="M248.1196,17.126 L254.2676,16.673 L254.9236,19.143 C256.4356,17.479 259.2076,16.018 265.2556,16.018 C273.8746,16.018 275.4866,20.805 275.4866,27.912 L275.4866,45.199 L267.6246,45.199 L267.6246,29.475 C267.6246,25.997 267.2216,23.477 262.4336,23.477 C256.9396,23.477 256.0316,26.047 256.0316,29.827 L256.0316,45.199 L248.1196,45.199 L248.1196,17.126 Z" id="N" stroke="none" fill="#221F20" fill-rule="evenodd"></path>
+  <path d="M280.355,30.8857 C280.355,22.9217 282.976,16.0177 295.172,16.0177 C307.723,16.0177 308.126,24.3837 307.874,33.9097 L288.016,33.9097 C288.469,37.3367 290.133,39.3527 295.223,39.3527 C299.609,39.3527 302.079,38.2947 304.195,36.7327 L308.177,41.8737 C304.497,44.6457 300.87,45.7537 295.223,45.7537 C282.876,45.7537 280.355,38.9997 280.355,30.8857 M288.016,28.2647 L300.667,28.2647 C300.465,24.9897 299.81,22.6197 294.92,22.6197 C290.032,22.6197 288.418,24.7867 288.016,28.2647" id="E" stroke="none" fill="#221F20" fill-rule="evenodd"></path>
+  <path d="M52.3237,16.2715 C31.2067,13.1085 16.4377,16.2715 16.4377,16.2715 C14.4707,28.3185 16.4377,37.6275 16.4377,37.6275 C27.5717,39.6675 37.9697,37.6275 37.9697,37.6275 C38.1897,34.7245 37.9697,30.1795 37.9697,30.1795 C28.6407,30.8715 23.7667,30.5085 23.7667,30.5085 C23.4877,26.4695 23.7667,23.3905 23.7667,23.3905 C35.1657,21.3825 45.1467,23.3905 45.1467,23.3905 C45.1467,23.3905 47.0087,33.7235 45.1467,44.7465 C44.8647,44.7735 26.6137,47.9305 9.2607,44.7465 C9.2607,44.7465 5.9857,29.2455 9.2607,9.1525 C30.1137,6.1065 48.3707,8.4295 52.4077,9.0235 C53.0487,5.2405 53.2667,1.6955 53.2667,1.6955 C28.5557,-2.3065 2.0837,2.0335 2.0837,2.0335 C2.0837,2.0335 -2.5983,26.9815 2.0747,51.8665 C2.0747,51.8665 20.9187,56.6715 52.3237,51.8585 C56.5447,34.6555 52.3237,16.2715 52.3237,16.2715" id="Logo" stroke="none" fill="#221F20" fill-rule="evenodd"></path>
+</svg>
```

Comment:
are all these `fill` attributes OK here? if they're written in line like that you can't override the color in CSS, I'm pretty sure.